### PR TITLE
noetic-devel

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Merge together with https://github.com/tue-robotics/tue_robocup/pull/1081

### CODE REVIEW

File: action_server/test/test_import.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import unittest
```
Comment (jlunenburg) [inline action_server/test/test_import.py:1]: Does this test run?

**Review by jlunenburg — COMMENTED** [review]
Don't have many comments. Imports are quite messy in general (no grouping etc.) but you could consider that out of the scope of this MR.

File: action_server/test/test_import.py (hunk 1 above)
Comment (MatthijsBurgh) [reply action_server/test/test_import.py:1]: Yes, it does

**Review by jlunenburg — APPROVED** [review]
As mentioned, I consider the imports a bit messy but I'll leave it up to you whether you want to clean it now or the next time we start working on this.

**Pull request merged**